docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1079,6 +1079,168 @@ These are not currently modeled.
 
+Household Surveys
+^^^^^^^^^^^^^^^^^
+
+There are many different types of household surveys that we might want to include 
+in the model. Therefore, this documentation reflects a general framework for 
+household surveys. The research team would then be responsible for providing the 
+additional inputs for an individual survey. 
+
+**When to Sample** 
+
+There are two types of sampling plans: 
+
+1. A new random sample of the population at a defined time interval (e.g., sample 5% of the households each month for a year)
+
+- The sample will be taken at set time intervals (monthly, annually) that the research team will communicate in terms of time steps 
+- The total duration of the survey will be specified 
+- Each subsequent sample is assumed to be independent of the prior samples 
+
+2. A longitudinal sample will sample the **same** population on defined time steps (e.g., sample the same 5% of households each month for 6 months) 
+
+- The sample will be taken at set time intervals (monthly, annually) that the research team will communicate in terms of time steps 
+- The total duration of the survey will be specified 
+- The sample will be taken and kept the same for all households 
+
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Home Address 
+  * - Home Zip Code 
+
+**Who to Sample** 
+
+For surveys, there is a much more significant amount of non-response bias 
+compared to the annual census. Participation will be determined in a two 
+step process. 
+
+**Step 1:** Households will be randomly selected for participation at a rate 
+predetermined by the researcher. The selection should be stratified by state, 
+but no other variables. This will be a random sample. 
+
+**Step 2:** Households will be chosen to be non-responders and removed from 
+the sample. This step will vary significantly based on the mode of the survey. 
+There are three possible modes: mail/online (assumed to be the same for this 
+model), telephone, and personal visits. 
+
+The below table includes the percent of responses for each mode of survey by 
+race/ethnicity. These will be used to find the non-response based on the mode 
+of the survey. 
+
+This data is based on the ACS which uses all of these methods and tracks 
+the percent of respondents to each. [Jackson_2007]_ Since the ACS has an order to 
+their survey modes (mail -> telephone -> personal visit), this data should 
+not be used out of that order (e.g., we can **not** assume that a telephone only 
+interview would have an 11.9% responce rate for white people). 
+
+For surveys with mail, telephone, and personal visits, the data below 
+will add to 100%. Since this is unrealistic, we assume that non-response 
+bias for this case matches the census, using the data in the section above. 
```

Comment:
- On my first read, I didn't understand which section you were referring to. I'd be more specific, maybe "using the rates in the Simulant Omission tables in the decennial census section above."
- I don't think it will matter very much because the census omission rate is so low, but wouldn't it make more sense to include the omission rate all the time (not only when a survey has all three modes)? After all, these are percentages of respondents.
- Using the Census omission rates seems very optimistic to me. The [ACS response rate](https://www.census.gov/acs/www/methodology/sample-size-and-data-quality/response-rates/), even before the pandemic, was ~95%, while the Census omission rate would give 99.76% at the population level. Given that the percentages of respondents are from ACS, it might be better to use an ACS number. [This paper](https://www.census.gov/content/dam/Census/library/working-papers/2002/acs/2002_Griffin_01.pdf) has ACS response rate by race/ethnicity, and the first link has it by state.

Reply:
Yeah, I think for surveys that don't follow the methodology ordering of mail -> telephone -> personal interview (such as the GSS which does face-to-face) we would need new percentages to show only telephone or only face-to-face. There are a lot of surveys and types of surveys that depending on what we would like might require some additional information to complete. 

For using the ACS vs census response rate. The census does the post-enumeration survey and so has better data on who is missed. I wasn't able to find data on who is missed from the ACS broken down by age/sex/race. If you have data (like what you sent but broken down more) I can swap it.

Reply:
Yes, ACS response rates are only available by state and year.

I understand why we would prefer age/sex/race stratified data, if we think that age/sex/race have significant effects on difficulty of PRL in a way that state does not. That said, it's quite confusing to model survey nonresponse while knowingly underestimating it by 20-80x (e.g. [CPS has an 80% response rate](https://www.bls.gov/osmr/response-rates/home.htm)). Maybe it doesn't matter much for PRL, but it would be good to at least add a limitation/note explaining why we did this and why it (counter-intuitively) doesn't matter too much. For example, applying an un-stratified ACS response rate adjustment would be equivalent to a different sample rate -- but then again, it's possible that the sample rate (and therefore data size) do matter for PRL applications.

Also, maybe I missed it, but did you respond to the second bullet point above?

Reply:
My reasoning for wanting age/sex/race specific data is that in real life, these things all overlap with stuff that makes PRL harder: 
- Having a higher probability for the same people to be missed (by age/sex/race attributes across observers) will provide an additional challenge since they might appear rarely and not have a match 
- Hispanic names can be longer and a combination of other names in the household 
- Larger family sizes or different family structures that are more difficult to match on taxes 
- Younger people are more likely to move, be in group quarters, or be missed in the census 
- Elderly people are also more likely to be in group quarters, and are less likely to be on taxes as they are not employed 

While I know not all of the above are currently in the sim, I was hoping to design something that would grow more complex with a more complex model. I think a state based missing rate or random missing rate would miss some of this. 

Yes, I am aware that ACS has a very high response rate. I think for adding additional specific survey observers, we might have to provide new data for increased missing folks. My hypothesis in creating this was that a lot of surveys would follow the order of the modes includes in ACS. So we could use this data for a mail/online only survey or for a mail and telephone only survey. This obviously does not apply to all surveys. 

The CPS, as with the GSS, does not follow this mode of survey pattern and so the data is not really applicable. The CPS does telephone and personal visits. Adding this specific survey would require some additional documentation. If there is a specific survey that should be added, we can do that. I will add to the limitations that the ACS has a high response rate. I have tried to clearly state that the data should be used when the modes match only. 

I will apply the census omission rate to all modes.